requirements.txt

```diff
@@ -12,3 +12,4 @@ dockerfile-parse~=0.0
 requests~=2.23
 stevedore>=1.32
 pbr>=5.4
+debut==0.9.4
```

Comment:
Are any of debut's dependencies overlapping with tern's? If no then please change this to `debut>=0.9` and if yes then please change it to `debut~=0.9`. Thanks!

Reply:
Done!